wal/wal.go

```diff
@@ -521,6 +521,14 @@ func (w *WAL) saveEntry(e *raftpb.Entry) error {
 	return nil
 }
 
+func (w *WAL) saveEntryOpt(data []byte) error {
```

Comment:
call this `saveEntry` and get rid of the old `saveEntry`? Probably not much of a reason to have both `saveEntryOpt` and `saveEntry` functions.